tchannel-core/src/main/java/com/uber/tchannel/schemes/RawResponse.java

```diff
@@ -89,6 +96,12 @@ public ByteBuf getArg3() {
         return arg3;
     }
 
+    public void release() {
```

Comment:
this should implement ByteBufHolder if you want to treat this as a reference counted object.